server.ts

```diff
@@ -380,6 +384,20 @@ app.get(
 // Partners
 app.get('/api/v1/partners/dashboard/stats', partnersDashboardStats.getDashboardStats);
 app.get('/api/v1/partners/:partner_id/dashboard/stats', partnersDashboardStats.getDashboardStatsByPartnerId);
+// Returns the location of the org centers of _all_ the partners
+// that have had at least one active student.
+// Filtering by date is possible.
+app.get(
+  '/api/v1/partners/dashboard/organization-centers-with-students',
```

Comment:
The naming convention so far has been to use underscore instead of dash in the endpoint names. Here, we can have `/api/v1/partners/dashboard/organization_centres`.

Reply:
done now :)